sql/wsrep_client_service.cc

```diff
@@ -347,6 +347,10 @@ void Wsrep_client_service::debug_crash(const char *crash_point
   DBUG_EXECUTE_IF(crash_point, DBUG_SUICIDE(););
 }
 
+void Wsrep_client_service::notify_state_change() {
```

Comment:
This is not a conflict resolution. I think it would be more clear to add it as a separate commit.